src/quo2/components/wallet/account_card/component_spec.cljs

```diff
@@ -38,6 +38,11 @@
       (h/render [account-card/view data])
       (h/is-truthy (h/get-by-text empty-username))))
 
+  (h/test "Renders Missing Keypair"
+    (let [data (get-test-data :missing-keypair false false false true)]
```

Comment:
it's hard to see what the test data is with this permutation of true/false. 
Could you refactor the function `get-test-data` and instead take a map? we can then just add `true` for the values we are setting and leave out the `false` values as it will default to `nil` anyway. will make it much easier to read these tests 👍

Reply:
Sure